src/protagonist/DLCS.AWS/SNS/Messaging/IBatchCompletedNotificationSender.cs

```diff
@@ -0,0 +1,16 @@
+﻿using DLCS.Model.Assets;
+
+namespace DLCS.AWS.SNS.Messaging;
+
+public interface IBatchCompletedNotificationSender
+{
+    /// <summary>
+    /// Broadcast multiple batch completed notification
+    /// </summary>
+    Task SendBatchCompletedMessages(IQueryable<Batch> completedBatches, CancellationToken cancellationToken = default);
```

Comment:
This shouldn't be possible - any action that can 'finish' a batch will be done on a single asset.